app/src/main/java/ca/deflector/passable_shots/MainActivity.java

```diff
@@ -44,7 +45,20 @@ public void onPictureTaken(byte[] data, Camera camera) {
                 // call encryption here
                 try {
                     byte[] encrypted = EncryptionSystem.encrypt(data);
-                    FileOutputStream fos = new FileOutputStream(Environment.getExternalStorageDirectory() + "/passable." + System.currentTimeMillis());
+                    // determine latest number
+                    int maxPassableFile = 0;
+                    for (File f : Environment.getExternalStorageDirectory().listFiles())
```

Comment:
A PR is not a place to discuss project-wide coding standards. Send me an email if you want to discuss them, but no, I don't like to add an extra line, it makes less code to fix in the screen.

If you approve the PR, merge it right away!